app/src/main/java/ch/blinkenlights/android/vanilla/PreferencesActivity.java

```diff
@@ -116,6 +116,8 @@ public void onCreate(Bundle savedInstanceState)
 		{
 			super.onCreate(savedInstanceState);
 			addPreferencesFromResource(R.xml.preference_audio);
+
+			ThemeHelper.setAccentColor(this.getActivity());
```

Comment:
could probably be done in super?

Reply:
No, because super is android internal stuff. The effort of creating a "middle-man-class" is not worth the effort imho